packit/api.py

```diff
@@ -952,13 +950,15 @@ def sync_release(
         dist_git_branch = (
             dist_git_branch or self.dg.local_project.git_project.default_branch
         )
-        # process version and tag parameters
-        if version and tag:
+        version = None
+        # process versions and tag parameters
+        if versions and tag:
             raise PackitException(
-                "Function parameters version and tag are mutually exclusive.",
+                "Function parameters versions and tag are mutually exclusive.",
             )
         if not tag:
-            version = version or self.up.get_latest_released_version()
+            # for now let's just pick the first one
```

Comment:
```suggestion
            # [FIXME] for now let's just pick the first one
```

just so it catches the eye :D